src/main/java/uk/gov/digital/ho/hocs/info/api/CaseActionsResource.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.digital.ho.hocs.info.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.info.api.dto.CaseActionTypeDto;
+
+import java.util.List;
+
+@RestController
+@Slf4j
+public class CaseActionsResource {
+
+    private final CaseActionsTypeService caseActionTypeService;
+
+    @Autowired
+    public CaseActionsResource(CaseActionsTypeService caseActionTypeService) {
+        this.caseActionTypeService = caseActionTypeService;
+    }
+
+    @GetMapping("/case-actions/{caseType}")
```

Comment:
Wonder if we could have a more 'RESTful' endpoint... `case/actions/{caseType}` or `casetype/{casetype}/actions`.. or similar?